composer.json

```diff
@@ -26,17 +26,18 @@
         }
     },
     "require": {
-        "php": "^7.1",
+        "php": "^7.3",
         "ocramius/package-versions": "^1.0",
         "psr/container": "^1.0",
-        "symfony/console": "^4.1 || ^5.0",
-        "symfony/event-dispatcher": "^4.0 || ^5.0"
+        "symfony/console": "^4.3 || ^5.0",
+        "symfony/event-dispatcher": "^4.0 || ^5.0",
+        "symfony/polyfill-php80": "^1.17"
```

Comment:
since required php is 7.3, add `polyfill-php74` as well?

Reply:
We are not using any PHP 7.4 feature (methods defined in 7.4+) so no reason to require it here. We are using some PHP 8.0 functions, that's why it has been added.

Reply:
No - I only brought in polyfill-php80 since I'm using the `get_debug_type()` function internally, and that function is provided in the upcoming v8.